adapters/ds/sinks/always_on_rtsp/config.py

```diff
@@ -66,6 +66,17 @@ def __init__(self):
 
         self.framerate = opt_config('FRAMERATE', '30/1')
         self.sync = opt_config('SYNC_OUTPUT', False, strtobool)
+        self.max_allowed_resolution = opt_config(
+            'MAX_ALLOWED_RESOLUTION',
```

Comment:
Do we need a word "allowed" here? `MAX_RESOLUTION` should be enough.

Reply:
Set the title according to the requirements in the task.

Reply:
@dorgun please change as Pavel proposes. It is a good one.

Reply:
Fixed it.